.github/workflows/dart_ci.yml

```diff
@@ -31,15 +31,15 @@ jobs:
         if: always() && steps.install.outcome == 'success'
 
       - name: Analyze project source
-        run: dart run dart_dev analyze
+        run: dart analyze
         if: always() && steps.install.outcome == 'success'
 
       - name: Run tests with ddc
-        run: dart run dart_dev test
+        run: dart test
```

Comment:
iirc, `dart test` actually uses dart2js by default for browser tests, so this change should be reverted. Was there a reason behind changing from `ddev test` to `dart test` here and to `build_runner test` below?